opentracing-example/src/main/scala/zio/telemetry/opentracing/example/config/Configuration.scala

```diff
@@ -13,11 +14,14 @@ object Configuration {
   }
 
   object Live extends Service {
-    val load: Task[Config] = Task.effect(ConfigSource.default.loadOrThrow[Config])
+    val load: Task[Config] = for {
+      source <- ZIO.fromEither(TypesafeConfigSource
+        .fromHoconFile(new java.io.File("opentracing-example/src/main/resources/application.conf")))
```

Comment:
Can we avoid this hard-coded path?

Reply:
Don't do this, zio-config and typesafe config can handle the default path easily. Take a look [here](https://github.com/zio/zio-redis/blob/master/example/src/main/scala/example/Main.scala#L23). Also, please do the following:

- rename `Config` to `AppConfig`
- try to remove `Configuration` (i.e. load from main, see zio-redis)
- extract  configuration classes from companion object into their own files

Reply:
Done.